src/NUnitFramework/framework/Attributes/ParallelScope.cs

```diff
@@ -59,26 +61,27 @@ public enum ParallelScope
 
         /// <summary>
         /// Descendants of the test may be run in parallel with one another.
-        /// Valid on assemblies and classes but not on methods.
+        /// Valid on assemblies and classes but not on non-parameterized methods.
         /// </summary>
         Children = 256,
+
         /// <summary>
         /// Descendants of the test down to the level of TestFixtures may be 
         /// run in parallel with one another. Valid on assemblies and classes
         /// but not on methods.
         /// </summary>
         Fixtures = 512,
+
         /// <summary>
         /// Mask used to extract all the flags that impact descendants of a 
         /// test and place them in the TestExecutionContext.
         /// </summary>
-
         [EditorBrowsable(EditorBrowsableState.Never)]
         ContextMask = Children + Fixtures,
 
         /// <summary>
         /// The test and its descendants may be run in parallel with others at
-        /// the same level. Valid on classes and methods but not assemblies.
+        /// the same level. Valid on classes, parameterized methods, and assemblies.
```

Comment:
I agree that it's confusing to say All is actually invalid for assemblies unless we follow that up by marking the assembly invalid.
This would be a good thing to do if we want the freedom to change what it means for assemblies in the future. E.g. maybe it could be a signal to a future runner to control the assembly-level parallelization?

Reply:
Of course, that goes for Self as well.

Reply:
@mikkelbu Do you mind leaving out assemblies until we figure out the answer?